### PR TITLE
Added RTF to master and datacut datset descriptions

### PR DESCRIPTION
**Pull request by marcuspp**
### Description of change
Updated the description fields for both master and datacut datasets so that they can contain rich text (like the reference datasets).

Screenshot 2020-03-09 at 14 34 57

### Checklist

* [ ] Have tests been added to cover any changes?
* [x] Has the [CHANGELOG](https://github.com/uktrade/data-workspace/blob/master/CHANGELOG.md) been updated?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: dataworkspace/dataworkspace/tests/test_security.py [hunk 1]
```diff
@@ -24,24 +27,47 @@ def test_baseline_content_security_policy(client):
...
+    full_url = reverse(url, args=args)
+    response = staff_client.get(full_url, follow=True)
+    script_src = get_src(response, 'script-src')
+    assert ("'unsafe-inline'" in script_src) is unsafe_inline_script
+
+    style_src = get_src(response, 'style-src')
+    assert "'unsafe-inline'" in style_src
+
+
+def get_src(response, policy_type):
```
Comment (samuelhwilliams) [inline dataworkspace/dataworkspace/tests/test_security.py]: Might be nice to tweak the name on this - maybe `get_csp_section`?

File: dataworkspace/dataworkspace/tests/test_security.py [hunk 2]
```diff
@@ -24,24 +27,47 @@ def test_baseline_content_security_policy(client):
...
-    dataset = factories.ReferenceDatasetFactory.create()
+    args = None
+    if 'change' in url:
+        if 'reference' in url:
+            dataset = factories.ReferenceDatasetFactory.create()
+        elif 'datacut' in url:
+            dataset = factories.DataSetFactory.create()
+        else:
+            dataset = factories.DataSetFactory.create(type=DataSetType.MASTER.value)
+        args = (dataset.id,)
```
Comment (samuelhwilliams) [inline dataworkspace/dataworkspace/tests/test_security.py]: These nested `if`s feel a bit iffy (🤦‍♂) in a test. They're probably fine. I tried an alternative but not sure I love it either - ends up taking up more lines.

```python
@pytest.mark.parametrize(
    'url, factory, unsafe_inline_script',
    (
        ('admin:datasets_referencedataset_add', None, True),
        (
            'admin:datasets_referencedataset_change',
            ReferenceDatasetFactory.create,
            True,
        ),
        ('admin:datasets_masterdataset_add', None, True),
        ('admin:datasets_masterdataset_change', DataSetFactory.create, True),
        ('admin:datasets_datacutdataset_add', None, True),
        (
            'admin:datasets_datacutdataset_change',
            partial(DataSetFactory.create, type=DataSetType.MASTER.value),
            True,
        ),
        ('admin:index', False),
    ),
)
def test_dataset_admin_pages_allow_inline_scripts_for_ckeditor_support(
    staff_client, url, factory, unsafe_inline_script
):
    args = None
    if factory:
        dataset = factory()
        args = (dataset.id,)
```

Comment (marcuspp) [reply dataworkspace/dataworkspace/tests/test_security.py]: Agreed it is abit. How about this instead https://github.com/uktrade/data-workspace/pull/511/commits/d9c0868a35067a44c4058a7dc0cda722af5e1bc0

Comment (samuelhwilliams) [reply dataworkspace/dataworkspace/tests/test_security.py]: Nice 👍

**Pull request merged**